lib/sentry/logger_backend.ex

```diff
@@ -93,6 +93,11 @@ defmodule Sentry.LoggerBackend do
           Sentry.capture_exception(reason, opts)
         end
 
+      {{_reason, stacktrace}, {_m, _f, args}} when is_list(stacktrace) and is_list(args) ->
+        # Cowboy stuff
+        # https://github.com/ninenines/cowboy/blob/master/src/cowboy_stream_h.erl#L148-L151
```

Comment:
Hey @mitchellhenke, will you be adding code to capture this case in Sentry as well?

Reply:
Mentioned in https://github.com/getsentry/sentry-elixir/issues/380#issuecomment-585900553, but the events are for the same crash, so reporting twice would be a duplicate 🙂

Reply:
Ah, got it. Thanks!